docs/edsl.html

```diff
@@ -111,7 +111,7 @@ <h3 id="max-over-axis">Max Over Axis</h3>
 \[
 \Large
   \textcolor{red}{O(n)}
-  \textcolor{green}{&gt;=}
+  \textcolor{green}{\gte}
```

Comment:
No, the `>=` max aggregation op is not a `≥`. (Also this doesn't render correctly (the command is `\geq` in TeX) -- but again, just switch back to `>=` as this is a max aggregation.)

Reply:
Oh, and in terms of color, the `>` should match the `max` (so green) and the `=` should match the `=` (so yellow)